docs/recipes/code-coverage.md

```diff
@@ -170,6 +170,28 @@ after_success:
 
 Your coverage report will then appear on coveralls shortly after Travis completes.
 
+## Integrating with AVA's Watch mode
+
+While AVA's watch mode [won't](https://github.com/avajs/ava/issues/1100) integrate with `nyc` directly out of the box, you can achieve that using [nodemon](https://github.com/remy/nodemon).
+
+First, install `nodemon` as a dev dependency:
+
+```bash
+$ npm install nodemon --save-dev
+```
+
+Then, create a new npm scripts entry, let's say, `watch`:
+
+```json
+"scripts": {
+	"coverage": "nyc report --reporter=text-lcov | coveralls",
+	"test": "standard && nyc --reporter=html --reporter=text ava",
```

Comment:
Done. Please review. I tested the separated setup on one of my libraries, seems fine on mine. Thank you, that's a new thing learned. 👍